src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6579,4 +6579,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_FunctionPointersCannotBeCalledWithNamedArguments" xml:space="preserve">
     <value>A function pointer cannot be called with named arguments.</value>
   </data>
+  <data name="WRN_UnusedRecordParameter" xml:space="preserve">
+    <value>Parameter '{0}' is unread. Did you forget to use it to initialize the property with that name?</value>
```

Comment:
>Did you forget to use it to initialize the property with that name? [](start = 38, length = 67)

I am not sure the second sentence is necessary.

Reply:
It's not necessary, but I think it's useful to guide users to the most likely solution. It helps for the primary scenario motivating this change.

---
In reply to: [512796721](https://github.com/dotnet/roslyn/pull/48895#discussion_r512796721) [](ancestors = 512796721)